conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
So if we'd allow a time duration to be specified, then we would have to enforce that it is a multiple of the raw interval. Do you think that makes sense? I'm not sure if it makes sense or not, as it might also just lead to unnecessary confusion if we error on a reorder window duration.